tool/jt.rb

```diff
@@ -2148,6 +2156,7 @@ def graph(*args)
       '--engine.MultiTier=false',
       '--engine.NodeSourcePositions',
       '--vm.Dgraal.PrintGraphWithSchedule=true',
+      '--vm.Dgraal.PrintBackendCFG=true',
```

Comment:
We should only do this when one wants the assembly, not by default, because it's a lot of output, it's creating an extra file, it might slow down dumping, etc.
How about an `--asm` option to do this and use `cfg2asm` only in that case?